--- a/lib/wx/configure.in  2012-12-06 15:02:45.000000000 +0100
+++ b/lib/wx/configure.in	2012-12-24 20:44:03.000000000 +0100
@@ -75,26 +75,6 @@
    CFLAGS="-Owx"
 fi
 
-## Check that we are in 32 bits mode on darwin 
-## (wxWidgets require that it currently uses 32-bits Carbon)
-## Otherwise skip building wxErlang
-AC_CHECK_SIZEOF(void *)
-
-case $ac_cv_sizeof_void_p-$host_os in
-    8-darwin*)
-	if test X"$WX_BUILDING_INSIDE_ERLSRC" != X"true" ; then
-		AC_MSG_ERROR([Can not use 64bits wxWidgets on Darwin])
-	else
-		echo "Can not combine 64bits erlang with wxWidgets on MacOSX, wx will not be useable" > ./CONF_INFO
-		WXERL_CAN_BUILD_DRIVER=false
-		AC_MSG_WARN([Can not combine 64bits erlang with wxWidgets on MacOSX, wx will not be useable])
-	fi
-	WXERL_CAN_BUILD_DRIVER=false
-	;;
-    *)
-	;;
-esac
-
 PTHR_CFLAGS="-D_THREAD_SAFE -D_REENTRANT"
 
 OBJC_CC=$CC