diff --git a/lib/wx/configure.in b/lib/wx/configure.in
index 12b4de6..bfdf714 100755
--- a/lib/wx/configure.in
+++ b/lib/wx/configure.in
@@ -160,7 +160,7 @@ case $host_os in
 	   fi
   	fi
   	CFLAGS="$USER_CFLAGS $MAC_MIN -Wno-deprecated-declarations"
-	CPPFLAGS="$CPPFLAGS -D_MACOSX $PTHR_CFLAGS"
+	CPPFLAGS="$CPPFLAGS -I/opt/X11/include -D_MACOSX $PTHR_CFLAGS"
 	;;
     mingw32)
 	CFLAGS="$CFLAGS -DWIN32 -DWINVER=0x0500 -D_WINDOWS -D_UNICODE -DUNICODE"
@@ -183,12 +183,12 @@ AC_SUBST(OBJC_CFLAGS)
 
 case $host_os in
     darwin*)
-	LDFLAGS="-bundle -flat_namespace -undefined warning -fPIC $LDFLAGS"
+	LDFLAGS="-bundle -flat_namespace -undefined warning -fPIC -L/opt/X11/lib -framework Appkit $LDFLAGS"
 	# Check sizof_void_p as future will hold 64bit MacOS wx
 	if test $ac_cv_sizeof_void_p = 4; then
 	    LDFLAGS="-m32 $LDFLAGS"
 	fi		   
-	GL_LIBS="-framework OpenGL"
+	GL_LIBS="-framework OpenGL -framework Appkit"
 	;;
     win32)
         LDFLAGS="-dll $LDFLAGS"
