diff --git a/lib/wx/configure.in b/lib/wx/configure.in
index 7b35fed..71e77d1 100755
--- a/lib/wx/configure.in
+++ b/lib/wx/configure.in
@@ -75,26 +75,6 @@ if test  X"$MIXED_CYGWIN_VC" == X"yes" -o X"$MIXED_MSYS_VC" == X"yes"; then
    CFLAGS="-Owx"
 fi
 
-## Check that we are in 32 bits mode on darwin 
-## (wxWidgets require that it currently uses 32-bits Carbon)
-## Otherwise skip building wxErlang
-AC_CHECK_SIZEOF(void *)
-
-case $ac_cv_sizeof_void_p-$host_os in
-    8-darwin*)
-	if test X"$WX_BUILDING_INSIDE_ERLSRC" != X"true" ; then
-		AC_MSG_ERROR([Can not use 64bits wxWidgets on Darwin])
-	else
-		echo "Can not combine 64bits erlang with wxWidgets on MacOSX, wx will not be useable" > ./CONF_INFO
-		WXERL_CAN_BUILD_DRIVER=false
-		AC_MSG_WARN([Can not combine 64bits erlang with wxWidgets on MacOSX, wx will not be useable])
-	fi
-	WXERL_CAN_BUILD_DRIVER=false
-	;;
-    *)
-	;;
-esac
-
 PTHR_CFLAGS="-D_THREAD_SAFE -D_REENTRANT"
 
 OBJC_CC=$CC
@@ -156,7 +136,7 @@ case $host_os in
 	   fi
   	fi
   	CFLAGS=$saved_CFLAGS
-	CPPFLAGS="$CPPFLAGS -D_MACOSX $PTHR_CFLAGS"
+	CPPFLAGS="$CPPFLAGS -I/opt/X11/include -D_MACOSX $PTHR_CFLAGS"
 	;;
     mingw32)
 	CFLAGS="$CFLAGS -DWIN32 -DWINVER=0x0500 -D_WINDOWS -D_UNICODE -DUNICODE"
@@ -183,7 +163,7 @@ case $host_os in
 	if test $ac_cv_sizeof_void_p = 4; then
 	    LDFLAGS="-m32 $LDFLAGS"
 	fi		   
-	GL_LIBS="-framework OpenGL"
+	GL_LIBS="-framework OpenGL -framework Appkit"
 	;;
     win32)
         LDFLAGS="-dll $LDFLAGS"
